apps/system/utils/security_level_cmd.c

```diff
@@ -0,0 +1,127 @@
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/ss_slot_index.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/prctl.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/********************************************************************************
+ * Private Functions
+ ********************************************************************************/
+static void show_usage(void) 
+{
+	printf("\nUsage : sec_level [OPTION]...\n");
+	printf("Set your security level high or low, or get the current security level status.\n\n");
+	printf("Options :\n");
+	printf("high		set security level 1(high)\n");
+	printf("		Your security level becomes high\n\n");
+	printf("low		set security level 0(low)\n");
+	printf("		Your security level becomes low\n\n");
+	printf("status		show current value of security level\n\n");
+}
+
+static void tash_security_level(int argc, char **args)
+{
+        security_handle hnd;
+	security_data ss_data = {NULL, 0};
+	security_error ret;
+
+	if (argc < 2) {
+		printf("Use [--help] option if you want to know how to use.\n");
+		return;
+	}
+
+	/* Show usage */
+	if (!strncmp(args[1], "--help", 7)) {
+		show_usage();
+		return;
+	}
+
+	/* Init security handler */
+	ret = security_init(&hnd);
+        if (ret != SECURITY_OK) {
+		printf("[security_init] error ret : %d\n", ret);
+		return;
+	}
+
+	/* Read 'security_level' variable from secure storage */
+	if (!strncmp(args[1], "status", 7)) {
+		ret = ss_read_secure_storage(hnd, SS_SLOT_PATH_SECURITY_LEVEL, 0, &ss_data);
+		if (ret != SECURITY_OK) {
+			printf("[ss_read_secure_storage] error ret : %d\n", ret);
+			return;
+		}
+
+		if (!strncmp(ss_data.data, "1", 2)) {
+			printf("security level : high\n");
+		} else if (!strncmp(ss_data.data, "0", 2)) {
+			printf("security level : low\n");
+		} else {
+			printf("Wrong value is written in security_level.\n");
+			printf("[ss_read_secure_storage] security_level data : %s\n", ss_data.data);
+		}
+		
+		return;
+	}
+
+	/* Change security_level according to user input */
+	ss_data.length = 2;
+	if (!strncmp(args[1], "high", 5)) {
+		ss_data.data = "1";
+	} else if (!strncmp(args[1], "low", 4)) {
+		ss_data.data = "0";
+	} else {
+		printf("Invalid argument.\n");
+		printf("Use [--help] option if you want to know how to use.\n");
+		return;
+	}
+	
+        /* Write 'security_level' variable to secure storage  */
+	ret = ss_write_secure_storage(hnd, SS_SLOT_PATH_SECURITY_LEVEL, 0, &ss_data);
+        if (ret != SECURITY_OK) {
+		printf("[ss_write_secure_storage] error ret : %d\n", ret);
+		return;
+	} 
+	
+	printf("Security level is set\n");
+
+#ifdef CONFIG_SECURITY_LEVEL
+	/* Apply changed value to the kerenl variable */
+	ret = prctl(PR_SET_SECURITY_LEVEL, getpid(), NULL);
+	if (ret != OK) {
+		printf("Fail to launch set_security_level. ret : %d\n", ret);
```

Comment:
1. launch? call?
2. As I left before, let's use the same format for the logs. You add `[ ]` at every fail logs. But not here.